deploy/runtime/ui/Battery Dispatch Automated FOM.txt

```diff
@@ -656,7 +656,7 @@ Battery Dispatch FOM Automated
 TextEntry
 
 0
-2642
+2567
```

Comment:
![image](https://user-images.githubusercontent.com/54449384/141400731-35c6d995-eb3a-4a33-bbf7-878d7c76258e.png)
The labels are a bit unclear is charging from clipped power supposed to be allowed for a custom time series?

Reply:
This is a bug, clipped power should be disabled for custom time series. I'll get this fixed in the next push.